Makefile

```diff
@@ -766,3 +766,11 @@ $(BINARY_DEPS_DIR)/linux-%:
 ifneq ($(FETCH_BINARIES_TARGETS),)
 .SECONDARY: $(call FULL_FETCH_BINARIES_TARGETS, $(FETCH_BINARIES_TARGETS))
 endif
+
+E2E_BINARY?=./bin/e2e.test
+TINKERBELL_HARDWARE_REQUIREMENTS?=test/e2e/tinkerbell_hardware_count.yaml
```

Comment:
I changed the name of the yaml file to all-caps in a previous PR, so let's update this too.